cranelift/codegen/src/isa/aarch64/lower_neon.isle

```diff
@@ -0,0 +1,38 @@
+
+;;;; Rules for `iadd` ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(rule (lower (has_type ty @ (dynamic_lane _ _) (iadd x y)))
+      (value_reg (vec_rrr (VecALUOp.Add) (put_in_reg x) (put_in_reg y) (vector_size ty))))
```

Comment:
The `value_reg` and `put_in_reg` should be unnecessary now thanks to implicit conversions, if you want to remove them!